src/components/sbb-notification/sbb-notification.tsx

```diff
@@ -0,0 +1,260 @@
+import {
+  Component,
+  ComponentInterface,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  JSX,
+  Method,
+  Prop,
+  State,
+  Watch,
+} from '@stencil/core';
+import { InterfaceTitleAttributes } from '../sbb-title/sbb-title.custom';
+import { i18nCloseNotification } from '../../global/i18n';
+import {
+  createNamedSlotState,
+  documentLanguage,
+  HandlerRepository,
+  languageChangeHandlerAspect,
+  namedSlotChangeHandlerAspect,
+} from '../../global/helpers';
+import { InterfaceNotificationAttributes } from './sbb-notification.custom';
+import { AgnosticResizeObserver as ResizeObserver } from '../../global/helpers/resize-observer';
+
```

Comment:
Normally we don‘t use enums but I also don‘t have any strong opinion against :-)